darshan-util/pydarshan/darshan/experimental/plots/plot_opcounts.py

```diff
@@ -96,16 +92,6 @@ def plot_opcounts(report, ax=None):
             stdio['STDIO_FLUSHES']
         ]
 
-    def as_csv():
-        text = ""
-        text += ','.join(labels) +  ',Layer' + "\n"
-        text += ','.join(str(x) for x in posix_vals) + ',POSIX' + "\n"
-        text += ','.join(str(x) for x in mpiind_vals) + ',MPIIND' + "\n"
-        text += ','.join(str(x) for x in mpicol_vals) + ',MPICOL' + "\n"
-        text += ','.join(str(x) for x in stdio_vals) + ',STDIO' + "\n"
-
-        return text
-
```

Comment:
@jakobluettgau are you "ok" with removing this? It isn't under test, but not sure if that means it is actually destined for removal.